File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        // 배열의 길이 비교
        if (arr1.length > arr2.length) {
            return 1;
        } else if (arr1.length < arr2.length) {
            return -1;
        } else {
            // 배열의 길이가 같을 경우, 원소의 합 비교
            int sum1 = 0;
            int sum2 = 0;
            
            for (int num : arr1) {
                sum1 += num;
            }
            
            for (int num : arr2) {
                sum2 += num;
            }
            
            if (sum1 > sum2) {
                return 1;
            } else if (sum1 < sum2) {
                return -1;
            } else {
                return 0;
            }
        }
    }
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    if len(arr1) > len(arr2):  # 배열 길이 비교
        return 1
    elif len(arr1) < len(arr2):
        return -1
    else:  # 길이가 같을 경우 합 비교
        return 1 if sum(arr1) > sum(arr2) else -1 if sum(arr1) < sum(arr2) else 0
